src/Workleap.Extensions.Mongo.Tests/MongoIndexerTests.Inheritance.cs

```diff
@@ -0,0 +1,59 @@
+using Microsoft.Extensions.DependencyInjection;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Driver;
+using Workleap.Extensions.Mongo.Indexing;
+
+namespace Workleap.Extensions.Mongo.Tests;
+
+public partial class MongoIndexerTests
+{
+    [Fact]
+    public async Task UpdateIndexesAsync_Ignores_Automatically_Inherited_Index_Provider()
```

Comment:
The logs displayed in the test also show that the index was created successfully (only once, nothing else):

![image](https://github.com/gsoft-inc/wl-extensions-mongo/assets/14242083/66a64063-9693-40df-994a-e5d81874d80e)